app/models/concerns/audit_extensions.rb

```diff
@@ -33,9 +34,29 @@ def ensure_taxonomies_not_escalated
   private
 
   def log_audit
-    Foreman::Logging.with_fields(self.audited_changes) do
-      Foreman::Logging.logger('audit').info { "#{self.action} event for #{self.auditable_type} with id #{self.auditable_id}" }
+    telemetry_increment_counter(:audit_records_created, 1, type: self.auditable_type)
+    audit_logger = Foreman::Logging.logger('audit')
+    return unless (self.audited_changes && audit_logger.info?)
+    self.audited_changes.each_pair do |attribute, change|
+      audited_fields = {
+        audit_action: self.action,
+        audit_type: self.auditable_type,
+        audit_id: self.auditable_id,
+        audit_attribute: attribute
+      }
+      if self.action == :update
```

Comment:
This should be `"update"`, not `:update`

Reply:
Damn, changed.